runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -1840,7 +1840,7 @@ typedef struct {
   void*         tgt_addr_v[MAX_CHAINED_PUT_LEN];
   c_nodeid_t    locale_v[MAX_CHAINED_PUT_LEN];
   void*         src_addr_v[MAX_CHAINED_PUT_LEN];
-  uint64_t      src_v[MAX_CHAINED_PUT_LEN];
+  char          src_v[MAX_CHAINED_PUT_LEN * MAX_UNORDERED_TRANS_SZ];
```

Comment:
I'm curious, why not `src_v[MAX_CHAINED_PUT_LEN][MAX_UNORDERED_TRANS_SZ]`?  That would allow 5917 and 5918 to refer to just `info->src_v[vi]` rather than `&info->src_v[vi*MAX_UNORDERED_TRANS_SZ]`.

Reply:
That's a good question, that's much cleaner. Changing to that.